modules/openapi-generator/src/main/resources/python/python-experimental/signing.mustache

```diff
@@ -0,0 +1,317 @@
+# coding: utf-8
+{{>partial_header}}
+from __future__ import absolute_import
+
+from datetime import timedelta
+from six.moves.urllib.parse import urlencode, urlparse
+import pem
+from Crypto.PublicKey import RSA, ECC
+from Crypto.Signature import PKCS1_v1_5, pss, DSS
+from Crypto.Hash import SHA256, SHA512
+from base64 import b64encode
+from email.utils import formatdate
+
+from {{packageName}}.configuration import HttpSigningConfiguration
+
+HEADER_REQUEST_TARGET = '(request-target)'
+HEADER_CREATED = '(created)'
+HEADER_EXPIRES = '(expires)'
+HEADER_HOST = 'host'
+HEADER_DATE = 'date'
+HEADER_DIGEST = 'Digest'
+HEADER_AUTHORIZATION = 'Authorization'
+
+SCHEME_HS2019 = 'hs2019'
+SCHEME_RSA_SHA256 = 'rsa-sha256'
+SCHEME_RSA_SHA512 = 'rsa-sha512'
+
+ALGORITHM_RSASSA_PSS = 'PSS'
+ALGORITHM_RSASSA_PKCS1v15 = 'PKCS1v15'
+
+ALGORITHM_ECDSA_MODE_FIPS_186_3 = 'fips-186-3'
+ALGORITHM_ECDSA_MODE_DETERMINISTIC_RFC6979 = 'deterministic-rfc6979'
+ALGORITHM_ECDSA_KEY_SIGNING_ALGORITHMS = {
+    ECDSA_MODE_FIPS_186_3,
+    ECDSA_MODE_DETERMINISTIC_RFC6979
+}
+
+class HttpSigningConfiguration(object):
+    """NOTE: This class is auto generated by OpenAPI Generator
+
+    Ref: https://openapi-generator.tech
+    Do not edit the class manually.
+
+    :param key_id: The identifier of the cryptographic key, when signing HTTP requests.
+        An 'Authorization' header is calculated by creating a hash of select headers,
+        and optionally the body of the HTTP request, then signing the hash value using
+        a private key which is available to the client.
+    :param private_key_path: The path of the file containing a private key,
+        when signing HTTP requests.
+    :param signing_scheme: The signature scheme, when signing HTTP requests.
+        Supported value is hs2019.
```

Comment:
Can you update this to list the three acceptabled values here?

Reply:
done.